runtime/convertValues_test.go

```diff
@@ -330,13 +472,15 @@ func TestImportValue(t *testing.T) {
 
 			t.Parallel()
 
-			actual, err := importValue(nil, tt.value, tt.expectedType)
+			inter := newTestInterpreter(t)
+
+			actual, err := importValue(inter, tt.value, tt.expectedType)
 
 			if tt.expected == nil {
 				require.Error(t, err)
 			} else {
 				require.NoError(t, err)
-				assert.Equal(t, tt.expected, actual)
+				AssertValuesEqual(t, inter, tt.expected, actual)
```

Comment:
`AssertValuesEqual` (and similarly named assertion helpers) test for equality using the value's `Equal()` function